templates/travel/partials/results/results.snip.html

```diff
@@ -0,0 +1,128 @@
+                  <div class="p1">
+                    <label class="travel-like">
+                      {{#liked}}<input type="checkbox" checked class="absolute"/>{{/liked}}
+                      {{^liked}}<input type="checkbox" class="absolute"/>{{/liked}}
+                      <div class="travel-like-hearts circle inline-block relative">
+                        <div class="travel-like-heart-tiny travel-like-heart-solid absolute"></div>
+                        <div class="travel-like-heart-tiny travel-like-heart-solid absolute"></div>
+                        <div class="travel-like-heart-tiny travel-like-heart-solid absolute"></div>
+                        <div class="travel-like-heart travel-like-heart-white absolute mx-auto "></div>
+                        <div class="travel-like-heart travel-like-heart-solid absolute mx-auto "></div>
+                        <div class="travel-like-heart travel-like-heart-outline absolute mx-auto "></div>
+                      </div>
+                    </label>
+                  </div>
+                </div>
+              </div>
+              <div class="h2 line-height-2 mb1">
+                <span class="travel-results-result-text">{{name}}</span>
+                <span class="travel-results-result-subtext h3">&bull;</span>
+                <span class="travel-results-result-subtext h3">$&nbsp;</span><span class="bold">{{price.value}}</span>
+              </div>
+              <div class="h4 line-height-2">
+                <!--
+                  Outputting SVG with amp-mustache is currently blocked. Plain
+                  glyphs are used here instead for the map icon and rating.
+                  see: https://github.com/ampproject/amphtml/issues/8214
+                -->
+                {{#reviews.count}}
+                  <div class="inline-block relative mr1 h3 line-height-2">
+                    <div>
+                      <span class="travel-img-icon-star-silver">
+                      </span><span class="travel-img-icon-star-silver">
+                      </span><span class="travel-img-icon-star-silver">
+                      </span><span class="travel-img-icon-star-silver">
+                      </span><span class="travel-img-icon-star-silver"></span>
+                    </div>
+                    <div class="absolute top-0">
+                      {{#reviews.averageRating.range}}<span class="travel-img-icon-star-green"></span>{{/reviews.averageRating.range}}
+                    </div>
+                  </div>
+                  <span class="travel-results-result-subtext mr1">{{reviews.count}} Reviews</span>
+                {{/reviews.count}}
+                {{^reviews.count}}
+                  <span class="travel-results-result-subtext mr1">Not Yet Reviewed</span>
+                {{/reviews.count}}
+                  <span class="nowrap">
+                    <span class="travel-img-icon-map-pin-outline-gray"></span>
```

Comment:
you will need to make this accessible , ideally add the image here using amp-img tag , or use a pseudo class for displaying the icon